DWM/update_abos_attributes_to_dwm.py

```diff
@@ -39,20 +39,24 @@ def get_netcdf_files(folder: str):
 
 
 def fix_abos_files(folder: str):
-    """ Call the functions to replace attributes that contain 'ABOS' and/or 'Australia Bluewater Observing System') """
+    """ Call the functions to replace attributes that contain 'ABOS' and/or 'Australia(n) Bluewater Observing System') """
     files = get_netcdf_files(folder)
     for file in files:
         print("Updating file: {}".format(file))
         logging.info("Updating file: {}".format(file))
         ncobj = nc.Dataset(file, 'a')
         abos_attributes = find_att_with_str(ncobj, 'ABOS')
         abos_blue_attributes = find_att_with_str(ncobj, 'Australia Bluewater Observing System')
+        abos_blue2_attributes = find_att_with_str(ncobj, 'Australian Bluewater Observing System')
         fix_abos_attributes(ncobj, abos_attributes, old='ABOS', new='DWM')
         fix_abos_attributes(ncobj, abos_blue_attributes, old='Australia Bluewater Observing System', new='Deep Water Moorings')
+        fix_abos_attributes(ncobj, abos_blue2_attributes, old='Australian Bluewater Observing System', new='Deep Water Moorings')
```

Comment:
This is totally fine, but starting to get a bit repetitive. If you had more than three cases to deal with, it would be worth generalising it by creating a dictionary of substitutions (old: new) you want to do, and then iterating through it...
```python
    substitutions = {'ABOS': 'DWM',
                     'Australia Bluewater Observing System': 'Deep Water Moorings',
                     'Australian Bluewater Observing System': 'Deep Water Moorings'
                     }
    for file in files:
        # ...
        for old, new in substitutions.items():
            attribute_list = find_att_with_str(ncobj, old)
            fix_abos_attributes(ncobj, attribute_list, old=old, new=new)
```
Also, the logging line for each attribute could be moved into the `fix_abos_attributes` function, so you don't have to explicitly do it here at all.
(And finally, you could also use regular expressions ([`re.search`](https://docs.python.org/3.6/library/re.html#re.search)/`re.sub`) instead of an exact sub-string match to combine the two variants of the full facility name)